data/tests/fwupdtool.sh

```diff
@@ -269,3 +310,6 @@ fi
 echo "Refresh remotes"
 run refresh --json
 rc=$?; if [ $rc != 0 ]; then error $rc; fi
+
+# cleanup fake UEFI system
+rm -rf /tmp/fwupd
```

Comment:
if you follow my `mktemp -d` suggestion, I think you can use a shell trap to clean up.

Reply:
I've learned something today! Fixed.